File: tbsmith/json.php.ssl1304.com/models/DBConnector.php
<?php

class DBConnector {

    function connect() {
        $host = '127.0.0.1';
        $user = 'root';
        $pass = 'root';
        $port = '3306';
        $dbname = 'fakeUsers';
        return new PDO("mysql:host=$host;port=$port;dbname=$dbname", $user, $pass);
    }

}

?>

File: tbsmith/python.ssl1304.com 3/models/DBConnector.py
#!/usr/bin/python

import MySQLdb


class DBConnector():

    def get_connection(self):
        self.db = MySQLdb.connect(host="127.0.0.1",
                                  port=3306,
                                  user="root",
                                  passwd="root",
                                  db="fakeUsers")

    def execute_statement(self, stmnt=''):
        self.get_connection()
        cursor = self.db.cursor()
        cursor.execute(stmnt)
        return cursor.fetchall()

    def get_random_user(self):
        result = self.execute_statement("SELECT * from address \
                                         ORDER BY rand() LIMIT 1")
        return result
